File: Java/uberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class uberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;

    public uberBlack(String license, Account driver,
            Map<String, Map<String, Integer>> typeCarAccepted,
            ArrayList<String> seatMaterial) {
        super(license, driver);
        this.seatMaterial = seatMaterial;
        this.typeCarAccepted = typeCarAccepted;
    }
}

File: PHP/uberBlack.php
<?php
require_once("./Car.php");
class uberX extends Car
{
    public $typeCarAccepted;
    public $seatMaterial;
    public function __construct($license, $driver, $typeCarAccepted, $seatMaterial)
    {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterial = $seatMaterial;
    }
}
